packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -223,12 +223,36 @@ export class SchemaFactory<
 	 * If each library exporting schema picks its own globally unique scope for its SchemaFactory,
 	 * then all schema an application might depend on, directly or transitively,
 	 * will end up with a unique fully qualified name which is required to refer to it in persisted data and errors.
-	 *
-	 * @param scope - Prefix appended to the identifiers of all {@link TreeNodeSchema} produced by this builder.
-	 * Use of [Reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) or a UUIDv4 is recommended to avoid collisions.
-	 * You may opt out of using a scope by passing `undefined`, but note that this increases the risk of collisions.
 	 */
-	public constructor(public readonly scope: TScope) {}
+	public constructor(
+		/**
+		 * Prefix appended to the identifiers of all {@link TreeNodeSchema} produced by this builder.
+		 * Use of [Reverse domain name notation](https://en.wikipedia.org/wiki/Reverse_domain_name_notation) or a UUIDv4 is recommended to avoid collisions.
+		 * You may opt out of using a scope by passing `undefined`, but note that this increases the risk of collisions.
+		 *
+		 * @remarks
+		 * Generally each library which is developed independently (possible a package, but could also be part of a package or multiple packages developed together) should get its own globally unique `scope`.
+		 * This ensures that the given library can name its schema (via the name parameter passed to the schema building methods, or implicitly for structurally named schema)
+		 * in a way that must only be unique within the library.
```

Comment:
I can't parse "library can name in a way that must be unique" part. The mix of "can" and "must" is weird to me (but maybe I do not get details of beautiful English language :))

I think you want to say that if two distinct types happen to have same runtime ID (which is schema factory ID + type ID) then really bad things would happen (maybe describe what happens and point to "help with debugging" article). In order for types to have unique runtime IDs, they need to ensure combination of factory schema ID + type ID is unique in a container.

Only after that intro, I'd give advice on usage of revers domain names - it's somewhat misplaced IMHO in the sequence.

Reply:
I have simplified the grammar and reordered the docs accordingly. Should be better now.